### ISSUE TITLE
Improperly escaped character (emoji) in video description breaks admin view.

### ISSUE THREAD

**Issue by just-max** [joschahenningsen/TUM-Live#442]
Adding an emoji to the video description breaks the course admin view. The GitHub JSON highlighter gives away the problem:
```json
{
   "color":"success",
   "courseId": 999,
   "description":"Tweedback: [https://tweedback.de/`XXXX`](https://tweedback.de/xxxx) 🐎

Lecturer: XXX XXX",
   "end":"XXX",
   "files":[
      {
         "friendlyName":"Default view",
         "id":999
      }
   ],
   "hasStats":true,
   "isConverting":false,
   "isLiveNow":false,
   "isPast":true,
   "isRecording":true,
   "lectureHallId":0,
   "lectureHallName":"XXX",
   "lectureId":999,
   "name":"Lecture 3: XXX",
   "seriesIdentifier":"",
   "start":"XXX",
   "streamKey":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
}
```

Adding an emoji is not in and of itself a problem (I added a couple with no issues), but the actual problem is the newline literal in the JSON value (which must be escaped as \n, and $something is presumably choking on the unicode emoji followed by a literal new line while trying to encode the description string as JSON).

This error occurs (client-side, of course) [here](https://github.com/joschahenningsen/TUM-Live/blob/11d1f2e6000ecec8954f1d62ac3b9444c66bf9b0/web/template/partial/course/manage/course-lecture-management.gohtml#L66). Specifically, the string in the code block above (more or less, I've formatted/removed any identifying information) is parsed returned from admin.decodeHtml and passed to JSON.parse here:
```text
new admin.Lecture(JSON.parse(admin.decodeHtml('{{.GetJson $lectureHalls}}')),'{{$course.Slug}}'),
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                             returns faulty JSON
```

#### Result:

After saving, a popup (?) appeared with the page source:

![image](https://user-images.githubusercontent.com/17551908/167495955-77f8b541-ea27-40a6-8b74-5e3e906cbcac.png)

Now, when loading the Admin page, 0 lectures are visible:

![image](https://user-images.githubusercontent.com/17551908/167497711-580f754c-19da-4f2d-a25c-8d05faf24113.png)

#### Relevant commits

https://github.com/joschahenningsen/TUM-Live/commit/ce6cc1406c0ae189e6cc0b2df4bd0535b53b5da6 probably

Browser: Firefox 99.0.1

<details>
<summary>Extra stuff, probably not relevant, duplicated above</summary>

I tried using adding the following to the description of a lecture:
```md
Slides covered: 0<span>&emdash;</span>0
```
 (I am aware it should actually be `mdash`, but I'm not sure if that's related)

(Also, I also had unsaved video sections, if that's relevant)

After saving, a popup (?) appeared with the page source:

![image](https://user-images.githubusercontent.com/17551908/167495955-77f8b541-ea27-40a6-8b74-5e3e906cbcac.png)

Now, when loading the Admin page, 0 lectures are visible:

![image](https://user-images.githubusercontent.com/17551908/167497711-580f754c-19da-4f2d-a25c-8d05faf24113.png)

The console shows a JSON parsing error (with a traceback of dubious utility):
```text
Uncaught SyntaxError: JSON.parse: bad control character in string literal at line 1 column 120 of the JSON data
    result https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js line 5 > AsyncFunction:25
    Te https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    Oe https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    Te https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    setTimeout handler*Te/< https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    Te https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    anonymous https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js line 5 > AsyncFunction:3
    dn https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    qt https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:1
    k https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    <anonymous> https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    D https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    r https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    n https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    tr https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    w https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    sr https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    sr https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    <anonymous> https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    VoidFunction* https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
    <anonymous> https://live.rbg.tum.de/static/node_modules/alpinejs/dist/cdn.min.js:5
cdn.min.js line 5 > AsyncFunction:25:49
```

The lectures all still exist and the VoDs can be watched as expected.

</details>

**Comment by joschahenningsen** [comment]
Nasty, Thanks for reporting. I rolled back to a previous version until this is resolved.

**Comment by joschahenningsen** [comment]
Deployed the fix #443, can you verify that everything looks good? :)
